core/server/common/src/test/java/alluxio/master/RegistryTest.java

```diff
@@ -31,11 +29,7 @@
 import javax.annotation.Nullable;
 
 public final class RegistryTest {
-
-  @Rule
-  public ExpectedException mThrown = ExpectedException.none();
```

Comment:
Thanks for getting rid of this with `Assert.assertThrows()`. Are there any other places which use this style that can be removed as well?

Reply:
I can look into more places that use this pattern.